src/webapp/pages/incident-action-plan/useIncidentActionPlan.ts

```diff
@@ -87,7 +90,15 @@ export function useIncidentActionPlan(id: Id) {
             incidentActionPlan => {
                 const incidentActionExists = !!incidentActionPlan?.actionPlan?.id;
                 const incidentActionOptions = incidentActionPlan?.incidentActionOptions;
-
+                const currentEventTracker = getCurrentEventTracker();
```

Comment:
I realized that this creates an infinite loop. `getCurrentEventTracker()` is called and also `changeCurrentEventTracker()` that changes the state value `currentEventTracker`, that is a dependency of `getCurrentEventTracker()` so a new instance is created and this function is a dependency of this `useEffect` creating the loop